src/script/main/globals.ts

```diff
@@ -45,6 +45,7 @@ import 'Components/toggle/InfoToggle';
 import 'Components/UserSearchableList';
 import '../Config';
 
+import '../page/AppMain';
```

Comment:
There are probably quite a few components that you can remove from the globals now (at least the `AppLock`, `RestrictedImage`, `RestrictedVideo`... and probably others)